src/python/gudhi/simplex_tree_multi.pyx

```diff
@@ -0,0 +1,970 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Vincent Rouvreau
+#
+# Copyright (C) 2016 Inria
+#
+# Modification(s):
+#	- 2022/11 Hannah Schreiber / David Loiseaux : adapt for multipersistence. 
+#   - YYYY/MM Author: Description of the modification
+
+
+
+__author__ = "Vincent Rouvreau"
+__copyright__ = "Copyright (C) 2016 Inria"
+__license__ = "MIT"
+
+
+from cython.operator import dereference, preincrement
+from libc.stdint cimport intptr_t
+from libc.stdint cimport uintptr_t
+from typing import Any
+
+cimport numpy as cnp
+import numpy as np
+cnp.import_array()
+
+cimport gudhi.simplex_tree_multi
+cimport cython
+from gudhi import SimplexTree ## Small hack for typing
+
+from filtration_domination import remove_strongly_filtration_dominated, remove_filtration_dominated
```

Comment:
This dependency needs to be documented. Does the whole file need to depend on it, or could we make it so only one function depends on it?

Reply:
This dependence is only needed in the `collapse_edge` method. I can put this dependence in this method indeed